bson/_cbsonmodule.c

```diff
@@ -617,9 +647,9 @@ int convert_codec_options(PyObject* self, PyObject* options_obj, codec_options_t
     if (type_marker < 0) {
         return 0;
     }
-
+    struct module_state *state = GETSTATE(self);
```

Comment:
Due to legacy reasons, local variables like this need to be defined at the top of the function.

Reply:
Makes sense. Changed.